Assets/00_Content/Scripts/Interactables/Interactable.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Interactables {
+	public class Interactable : MonoBehaviour, IInteractable {
+
+		public Action onInteraction;
+		public Action onInteractionCancelled;
```

Comment:
Wouldn't having these be UnityEvents be more flexible? Otherwise you should probably add the `event` keyword.

Reply:
It´s getting scrapped, no reason for it after overriding virtual methods in baseclass. But good reminder about the "event" keyword.. I always forget.